src/renderer/components/RandomizerTile/LuckyBeggarList.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <ul class="wrapper" v-if="luckyBeggarList.length > 0">
+    <li class="field" v-for="(item, index) in getLastRecords(luckyBeggarList)" v-bind:key="index">
+      <b-tag
+        :type="index === 0 ? 'is-success' : 'is-light'"
+        :size="index === 0 ? 'is-medium' : ''"
+      >{{item}}</b-tag>
+    </li>
+    <li v-if="luckyBeggarList.length > lastRecordNumber">...</li>
+  </ul>
+</template>
+
+<script>
+export default {
+  props: {
+    luckyBeggarList: { type: Array },
+    lastRecordNumber: {
+      type: Number,
+      default: 4,
+    },
+  },
+  methods: {
+    getLastRecords() {
```

Comment:
This method can be a computed property, so instead of being in the `methods` object, it can go into `computed` object. The difference is caching - computed properties are cached as long as their dependencies don't change.

Reply:
And computed properties can't have arguments passed into.